app/src/main/java/com/codesoom/assignment/application/UserService.java

```diff
@@ -0,0 +1,75 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.UserNotFoundException;
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.domain.UserRepository;
+import com.codesoom.assignment.dto.UserData;
+import org.springframework.stereotype.Service;
+
+/**
+ * 사용자 정보를 등록,수정,삭제하는 서비스 로직입니다.
```

Comment:
`서비스 로직입니다`는 불필요합니다. 이 정도면 적당하겠죠. 물론 더 좋은 표현도 있을 거에요.
```suggestion
 * 사용자 정보 등록, 수정, 삭제를 담당합니다.
```

Reply:
네 좀 주석으로 표현하는게 좀 쉽지않네요